docs/articles/documentation/reference/testcafe-api/runner/compileroptions.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: docs
+title: Runner.compilerOptions Method
+permalink: /documentation/reference/testcafe-api/runner/compileroptions.html
+---
+# Runner.compilerOptions Method
+
+Specifies test compilation settings. At the moment, only configures the [TypeScript compiler](../../../guides/concepts/typescript-and-coffeescript.md#customize-compiler-options).
```

Comment:
> At the moment, only configures the [TypeScript compiler](../../../guides/concepts/typescript-and-coffeescript.md#customize-compiler-options).

As far as I remember, we prefer other expressions rather than `at the moment` when we speak about currently implemented functionality: *currently*, *in the current version*...